source/Halibut/HalibutRuntime.cs

```diff
@@ -264,6 +265,36 @@ async Task<ResponseMessage> SendOutgoingRequestSynchronouslyAsync(RequestMessage
             return response;
         }
 
+        async Task<ResponseMessage> SendOutgoingRequestAsync(RequestMessage request, MethodInfo methodInfo, CancellationToken cancellationToken)
```

Comment:
I think I added this method [here](https://github.com/OctopusDeploy/Halibut/pull/356/files#diff-a5f79eb60063621324c5189327912f06a77a5ade175b6ce22430447ee41a60e4R278) I think that PR ought to get in asap.

Reply:
Yerp, need an async entry point to get this path tested!